sa/db-next/boulder_sa/20240514000000_Paused.sql

```diff
@@ -0,0 +1,16 @@
+-- +migrate Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+CREATE TABLE `paused` (
```

Comment:
I recall that we previously decided that this table should not have an auto-increment ID column, but I no longer fully remember the reasoning why. Can we document that reasoning here for future readers?